.gitignore

```diff
@@ -1,2 +1,3 @@
+idea/
```

Comment:
This is specific to your development environment and unrelated to MathQuill, please add this to your global `.gitignore` instead: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore